### PR TITLE
Add to occupations readme

### PR DESCRIPTION
**Pull request by lizgzil**
---

# Description

SOCMapper docs! I should add some emojis though?

# Instructions for Reviewer

In order to test the code in this PR you need to ...

Please pay special attention to ...

# Checklist:

- [ ] I have refactored my code out from `notebooks/`
- [ ] I have checked the code runs
- [ ] I have tested the code
- [ ] I have run `pre-commit` and addressed any issues not automatically fixed
- [ ] I have merged any new changes from `dev`
- [ ] I have documented the code
  - [ ] Major functions have docstrings
  - [ ] Appropriate information has been added to `README`s
- [ ] I have explained this PR above
- [ ] I have requested a code review

### CODE REVIEW

File: dap_prinz_green_jobs/pipeline/green_measures/occupations/README.md [hunk 1]
```diff
@@ -1,35 +1,119 @@
...
+| Assistant nurse                      | 6131/99           | Nursing auxiliaries and assistants n.e.c. | Nursing auxiliaries and assistants           | 6141          | assistant nurse      |
+| Senior financial consultant - London | 2422/02           | Financial advisors and planners           | Finance and investment analysts and advisers | 3534          | financial consultant |
+
+## Methodology
+
+The SOCMapper works by finding the semantically closest job titles between the inputed job titles and the job titles in the ONS SOC dataset. An overview of the methodology is in the diagram below.
+
+<p align="center">
+  <img src="SOCMapper_overview.jpg" />
+</p>
```
Comment (india-kerle) [inline dap_prinz_green_jobs/pipeline/green_measures/occupations/README.md:59]: love this

File: dap_prinz_green_jobs/pipeline/green_measures/occupations/README.md [hunk 2]
```diff
@@ -1,35 +1,119 @@
...
 
-If the names of the SOC codes aren't needed then you can set `return_soc_name=False`. The variables `soc_mapper.soc_2020_6_dict` and `soc_mapper.soc_2020_4_dict` give the names of each SOC 2020 6 and 4 digit codes.
+```
+['Data Scientist', 'Electric motor assembler', 'Data visualisation developer']
+```
+
+**Step 2:** We process the [ONS SOC dataset](https://www.ons.gov.uk/methodology/classificationsandstandards/standardoccupationalclassificationsoc/standardoccupationalclassificationsocextensionproject). An example of this dataset is:
+
+| SOC 2010 | SOC 2020 | SOC 2020 Ext Code | INDEXOCC           | ADD       | IND         | INDEXOCC NATURAL WORD ORDER | SOC 2020 UNIT GROUP DESCRIPTIONS                   | SUB-UNIT GROUP DESCRIPTIONS                               |
+| -------- | -------- | ----------------- | ------------------ | --------- | ----------- | --------------------------- | -------------------------------------------------- | --------------------------------------------------------- |
```
Comment (india-kerle) [inline dap_prinz_green_jobs/pipeline/green_measures/occupations/README.md:78]: really helpful to have an example

File: dap_prinz_green_jobs/pipeline/green_measures/occupations/README.md [hunk 3]
```diff
@@ -1,35 +1,119 @@
...
+
+|                              | ONS job title matched to | SOC     | SOC description                                 |
+| ---------------------------- | ------------------------ | ------- | ----------------------------------------------- |
+| Data Scientist               | data scientist           | 2433/02 | Data scientists                                 |
+| Electric motor assembler     | motor assembler electric | 8141/00 | Assemblers (electrical and electronic products) |
+| Data visualisation developer | None                     | None    | None                                            |
+
+However, if we had set slightly different conditions for the consensus approach, another outcome could be that the "Data visualisation developer" job title was mapped to the SOC "2134 - Programmers and software professionals" since 2 out of the 4 matches with over 0.45 similarity were from this 4-digit SOC.
+
+# Occupational Green Measures
```
Comment (india-kerle) [inline dap_prinz_green_jobs/pipeline/green_measures/occupations/README.md]: not here but i would add an additional line under usage to say that thats to map to green measure

**Review by india-kerle — APPROVED** [review]
looks great! Love the image also - i'm so committed to emojis but it's not a must :)

File: dap_prinz_green_jobs/pipeline/green_measures/occupations/README.md (hunk 3 above)
Comment (lizgzil) [reply dap_prinz_green_jobs/pipeline/green_measures/occupations/README.md]: done!

**Pull request merged**